CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Framer Motion adheres to [Semantic Versioning](http://semver.org/).
 
 Undocumented APIs should be considered internal and may change without warning.
 
+### Updated
+
+-   Refactored `motion` component features to load without React components.
+
+### Removed
+
+-   Fallback support for `IntersectionObserver`.
```

Comment:
We currently run `IntersectionObserver` without polyfill on Framer sites so this is safe for us to do.